e2e/wdio/platform/tests/combobox.e2e-spec.ts

```diff
@@ -0,0 +1,63 @@
+import { webDriver } from '../../driver/wdio';
+import { ComboBoxPo } from '../pages/combobox.po';
+import ComboboxData from '../fixtures/appData/combobox.page-content';
+
+describe('Combobox test suite', function() {
+    const comboBoxPage: ComboBoxPo = new ComboBoxPo();
+
+    beforeAll(() => {
+        comboBoxPage.open();
+    });
+
+    afterEach(() => {
+        webDriver.refreshPage();
+    });
+
+    it('Verify each combobox consist of input and button', () => {
+        comboBoxPage.verifyComboBoxComponents(ComboboxData.activeTypeNames, ComboboxData.notActiveTypeNames);
```

Comment:
this is over encapsulation, spec files should be readable in terms of `AAA`
where Arange is `open` in 90% of its

Reply:
methods should makes sense only if they are used several times mostly